notify/notify.go

```diff
@@ -228,6 +233,13 @@ func ActiveTimeIntervalNames(ctx context.Context) ([]string, bool) {
 	return v, ok
 }
 
+// RouteID extracts a RouteID from the context. Iff none exists, the
+// // second argument is false.
```

Comment:
```suggestion
// RouteID extracts a RouteID from the context. If none exists, the
// second argument is false.
```

The other comments have the same typo of `iff`

Reply:
Oh iff is not a spelling mistake, it refers to if and only if.

Reply:
Today I learned 🤯